app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Tab.kt

```diff
@@ -0,0 +1,396 @@
+      override val rowShape: Shape
+        @Composable
+        @ReadOnlyComposable
+        get() = MediumTabTokens.ContainerShape.value
+      override val tabShape: Shape
+        @Composable
+        @ReadOnlyComposable
+        get() = MediumTabTokens.TabShape.value
+    }
+
+    data object Large : TabSize() {
+      override val rowPadding: PaddingValues
+        get() = PaddingValues(
+          horizontal = LargeTabTokens.RowHorizontalPadding,
+          vertical = LargeTabTokens.RowVerticalPadding,
+        )
+      override val tabPadding: PaddingValues
+        get() = PaddingValues(
+          top = LargeTabTokens.TabTopPadding,
+          bottom = LargeTabTokens.TabBottomPadding,
+          start = LargeTabTokens.TabHorizontalPadding,
+          end = LargeTabTokens.TabHorizontalPadding,
+        )
+      override val textStyle: TextStyle
+        @Composable
+        @ReadOnlyComposable
+        get() = LargeTabTokens.TextFont.value
+      override val rowShape: Shape
+        @Composable
+        @ReadOnlyComposable
+        get() = LargeTabTokens.ContainerShape.value
+      override val tabShape: Shape
+        @Composable
+        @ReadOnlyComposable
+        get() = LargeTabTokens.TabShape.value
+    }
+  }
+
+  sealed class TabStyle {
+    @get:Composable
+    internal abstract val colors: TabColors
+
+    data object Default : TabStyle() {
+      override val colors: TabColors
+        @Composable
+        get() = with(HedvigTheme.colorScheme) {
+          remember(this) {
+            TabColors(
+              tabRowBackground = Color.Transparent,
+              chosenTabBackground = fromToken(ButtonSecondaryResting),
+              notChosenTabBackground = Color.Transparent,
+              textColor = fromToken(TextPrimary),
+            )
+          }
+        }
+    }
+
+    data object Filled : TabStyle() {
+      override val colors: TabColors
+        @Composable
+        get() = with(HedvigTheme.colorScheme) {
+          remember(this) {
+            TabColors(
+              tabRowBackground = fromToken(SurfacePrimary),
+              chosenTabBackground = fromToken(ButtonSecondaryAltResting),
+              notChosenTabBackground = Color.Transparent,
+              textColor = fromToken(TextPrimary),
+            )
+          }
+        }
+    }
+
+    data class TabColors(
+      val tabRowBackground: Color,
+      val chosenTabBackground: Color,
+      val notChosenTabBackground: Color,
+      val textColor: Color,
+    )
+  }
+}
+
+@Composable
+private fun TabItem(
+  onClick: () -> Unit,
+  text: String,
+  textStyle: TextStyle,
+  tabTextColor: Color,
+  contentPadding: PaddingValues,
+  onTextMeasured: (width: Int) -> Unit,
+  onHeightMeasured: (height: Int) -> Unit,
+  modifier: Modifier = Modifier,
+) {
+  HedvigText(
+    modifier = modifier
+      .clickable(
+        interactionSource = remember { MutableInteractionSource() },
+        indication = null, // as the sliding indicator is enough, I think
```

Comment:
Yeah since this is in reality an iOS component, I don't think they got an indicator there at all, so keep it like this and we can remove the comment too